Source/WebKit/WebProcess/WebPage/WebPage.cpp

```diff
@@ -1251,6 +1251,7 @@ EditorState WebPage::editorState(ShouldPerformLayout shouldPerformLayout) const
     if (result.selectionIsRange) {
         auto selectionRange = selection.range();
         result.selectionIsRangeInsideImageOverlay = selectionRange && ImageOverlay::isInsideOverlay(*selectionRange);
+        result.selectionIsInAutoFilledAndViewableField = selection.isInAutoFilledAndViewableField();
```

Comment:
(On second glance, I would probably call this `selectionIsRangeInAutoFilledAndViewableField`, to be more exact)